open_discussions/settings.py

```diff
@@ -407,6 +407,10 @@
     'JWT_AUTH_HEADER_PREFIX': 'Bearer',
 }
 
+EXTERNAL_LOGIN_URL = get_string('MICROMASTERS_EXTERNAL_LOGIN_URL', None)
```

Comment:
sorry, should have been more specific! I was suggesting we call the setting something like `MM_EXTERNAL_LOGIN_URL` as well as the environment variable